### PR TITLE
add .driveignore functionality

### LINKED ISSUE #9: add .driveignore functionality
add functionality that allows users to specify to prevent certain files from getting uploaded

### PR DESCRIPTION
**Pull request by raghav-dalmia**
There are two ways to untrack a file/folder. First one is create a `.driveignore` file in the tracked directory and then add name of files to be untracked.

Second is to use `drive ignore` command. Enter the file names(with extensions) as an argument. We can use `-l` flag to list out all the untracked files.

### OUTPUT
------

Screenshot from 2019-04-05 03-32-00

Screenshot from 2019-04-05 03-32-03

Screenshot from 2019-04-05 03-34-43

Screenshot from 2019-04-05 03-42-25

Fixes #9 
@nurdtechie98 @inishchith please review this PR.

### CODE REVIEW

File: drive_cli/actions.py [hunk 1]
```diff
@@ -447,7 +447,7 @@ def history(date, clear):
                 for date in History:
                     history = History[date]
                     for i in history:
-                        click.secho(date + "  " + i, fg='yellow', bold=True)
+                        click.secho(date + "  " + i, fg='yellow', bold=True
```
Comment (inishchith) [inline drive_cli/actions.py]: @raghav-dalmia Did you try to run this locally? you've missed to close with `)`.

**Comment by nurdtechie98** [comment]
@raghav-dalmia just make that tiny rest looks good to merge. Awesome Job 🎉

**Comment by shivam1708** [comment]
Great Job @raghav-dalmia

File: drive_cli/actions.py (hunk 1 above)
Comment (raghav-dalmia) [reply drive_cli/actions.py]: Yes, I run this on local. This might happens when I was resolving merge conflicts. Sorry about that.

**Comment by raghav-dalmia** [comment]
@nurdtechie98 @inishchith @shivam1708 I have made the changes, please review this PR.

**Comment by nurdtechie98** [comment]
@inishchith I'll squash n merge it for now.

**Pull request merged**